### ISSUE TITLE
App crashes on second start

### ISSUE THREAD

**Issue by robinbonnes** [cobrowseio/cobrowse-sdk-android-binary#1]
Thanks for this great tool. Its really going to help our customers.

However, when I launch a second session, the app crashes:

```
E/AndroidRuntime(28530): FATAL EXCEPTION: main
E/AndroidRuntime(28530): Process: nl.bundle.identifier, PID: 28530
E/AndroidRuntime(28530): java.lang.NullPointerException: Attempt to invoke virtual method 'android.media.Image android.media.ImageReader.acquireLatestImage()' on a null object reference
E/AndroidRuntime(28530): 	at io.cobrowse.core.FullDeviceScreenCapture.capture(FullDeviceScreenCapture.java:55)
E/AndroidRuntime(28530): 	at io.cobrowse.core.FrameLoopManager.captureFor(FrameLoopManager.java:97)
E/AndroidRuntime(28530): 	at io.cobrowse.core.FrameLoop.takeScreenshot(FrameLoop.java:82)
E/AndroidRuntime(28530): 	at io.cobrowse.core.FrameLoop.updateFrame(FrameLoop.java:105)
E/AndroidRuntime(28530): 	at io.cobrowse.core.FrameLoop.loop(FrameLoop.java:61)
E/AndroidRuntime(28530): 	at io.cobrowse.core.FrameLoop.access$100(FrameLoop.java:15)
E/AndroidRuntime(28530): 	at io.cobrowse.core.FrameLoop$1.run(FrameLoop.java:51)
E/AndroidRuntime(28530): 	at android.os.Handler.handleCallback(Handler.java:754)
E/AndroidRuntime(28530): 	at android.os.Handler.dispatchMessage(Handler.java:95)
E/AndroidRuntime(28530): 	at android.os.Looper.loop(Looper.java:163)
E/AndroidRuntime(28530): 	at android.app.ActivityThread.main(ActivityThread.java:6393)
E/AndroidRuntime(28530): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(28530): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:933)
E/AndroidRuntime(28530): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:823)
```

By the way, I've created a Flutter plugin, let me know what you think: https://github.com/robinbonnes/cobrowseio_flutter

**Comment by headlessme** [comment]
Thanks for reporting. We're working on a fix for this now. Can you confirm this only happens after using the full device capture mode?

**Comment by headlessme** [comment]
This should now be fixed in 1.10.3. Please re-open if the issue persists!

**Comment by robinbonnes** [comment]
Yes, I can confirm this is fixed. Thanks!
